File: 0791-custom-sort-string/0791-custom-sort-string.cpp
class Solution {
public:
    string customSortString(string order, string s) {
        string a;
        for (char i:order){
            if (s.find(i)!=string::npos){
                int x = count(s.begin(),s.end(),i);
                a.append(x,i);
            }
        }
        for (char j:s){
            if (find(order.begin(),order.end(),j)==order.end()){
                a+=j;
            }
        }
        return a;
    }
};

File: 0791-custom-sort-string/0791-custom-sort-string.py
class Solution:
    def customSortString(self, order: str, s: str) -> str:
        a=""
        for i in order:
            if i in s:
                x=s.count(i) 
                a+=i*x
        for i in s:
            if i not in order:
                a+=i 
        return a
